openslides/core/views.py

```diff
@@ -249,6 +249,11 @@ def activate_elements(self, request, pk):
             raise ValidationError({'detail': 'Data must be a list.'})
 
         projector_instance = self.get_object()
+        # reset scroll level
```

Comment:
This leads to not wanted behaviour:
The `activate_elements` is (I think only) used by stable projector elements. So the scroll level resets, if one projects a message or the current list of speakers overlay.
For the most non stable elements we use a combination of `clear_elements` and `prune_elements` (E.g. see in motions/base.js line 653 `project()`).
So just resetting the scroll level for pruning elements should be enough, or do you have a case where `activate_elements` should lead to a resetted scroll level?

Reply:
No, I was not sure if `active_elements` is used... I'll change it.

Reply:
Done. Reset scroll level was used in `prune_elements` only.